docs/sql-railroad.svg

```diff
@@ -0,0 +1,855 @@
+<?xml version="1.0" standalone="no"?>
+
+<!--
+Generated by putting the grammar below in
+https://tabatkins.github.io/railroad-diagrams/generator.html
+and making the following modifications to the SVG output pane:
+* preserving this comment and the XML prologue above
+* adding xmlns="http://www.w3.org/2000/svg" to the <svg> tag
+* adding <defs><style>...</style></defs> as the first child
+  of the <svg> element with the contents of
+  https://raw.githubusercontent.com/tabatkins/railroad-diagrams/gh-pages/railroad-diagrams.css
```

Comment:
The code review tools doesn't show image sources by default so see this if you want the diagram's derivation.